mlir/utils/jenkins/static-checks/clang-tidy.ignore

```diff
@@ -4,4 +4,5 @@ mlir/examples/standalone
 mlir/test
 mlir/utils/performance/ck-benchmark-driver
 build/
+external
```

Comment:
The suggestion was that you go manually fire off a build of the CI with `ignoreExternalLinting` checked.

Reply:
Ok. reverted the change